web/pandas/community/blog/asv-pandas-grant.md

```diff
@@ -0,0 +1,133 @@
+### Our experience
+For this grant, we had a fantastic Mentor, Marc Garcia. He was always
+willing to share his knowledge, explain unclear concepts and share helpful
+feedback. Whenever we would implement that feedback, it felt easier to work
+on more issues faster. We felt the growth from the time we started on this
+project, and we will carry it along as we contribute to more Open Source
+projects; this all goes back to Marc for his fantastic mentorship. It is
+also important to note that we received feedback from other contributors,
+stakeholders, and core devs during this process, which gave us a broader
+look at the work in Open Source projects.
+
+We also built a strong teamwork partnership. We helped each other a lot as
+we had numerous one-on-one calls to understand the tasks better. We always
+looked for ways to support each other from the technical side and encouraged
+each other when needed. For us, it was professional and human growth.
+
+### Running an Open Source Software Sprint
+The knowledge and experience acquired in this process allowed us to
+organize two virtual sprints. The events were carried out in the company
+of local PyLadies communities; the first one was on February 26th with
+[PyLadies Kampala](https://twitter.com/pyladieskla?s=11) and on March 21
+with [PyLadies Colombia](https://bit.ly/sprint-asv).
+
+While organizing these events, we learned how to organize and conduct a
+virtual Sprint. Some participants in the sprint ultimately had no idea
+about open source, and it was great explaining open source concepts and
+taking them through the Git workflow. Finally, they were able to make their
+first contribution. We learned how to follow up on contributors, helping
+them along the way until their PRs were merged and by reviewing their
+contributions on Github.
+
+The most outstanding achievement was mentoring new contributors and
+sharing the knowledge acquired from this grant with others participants
+in our respective communities. Most new contributors after the experience
+have gone ahead to apply for outreach and the upcoming google summer of
+code to apply the skills they learned from these sprints.
+
+### Conclusion
+In conclusion, we learned a lot from this experience from the code part,
+the workflow on the open-source projects, how to be resilient in difficult
+moments, and encouraging more women and people from our local communities
+to contribute to Open Source projects.
+
+Finally, if you want to be part of an open-source project, we invite you
+to check out GitHub repos for different projects you are interested in and
+search for the easy issues to work on and get started. Also, you can contact
+the maintainers of the projects with specific questions, search for the
+open-source communities in your country or contact us for more help.
+
+### Acknowledgments
+Many thanks to [NumFOCUS](https://numfocus.org/) for giving us this support
+through small grants and Marc for the excellent mentoring he generously
```

Comment:
There is a website for the small development grants in the NumFOCUS site, not sure if here or at the beginning, but probably useful to link to it.